doc/manual/nuc_handbook.rst

```diff
@@ -0,0 +1,257 @@
+
+This handbook assumes you are starting with a running system using the `defenestrate <https://git.m-labs.hk/M-Labs/defenestrate>`_ NixOS configuration for ARTIQ/Sinara. Notably, if you've recently ordered an ARTIQ/Sinara crate from M-Labs together with an accompanying preinstalled NUC, **this will be how your NUC is already set up.** Welcome! What follows is a short guide to the configuration itself, how to adapt it to your needs, and how to work with Nix and NixOS in general.
+
+.. note::
+
+    If you did not purchase a NUC from M-Labs, you can nonetheless exactly reproduce the environment described here by installing NixOS and adopting the same configuration files. (This ability to precisely replicate environments between machines is one of NixOS' great advantages.) See again the `defenestrate <https://git.m-labs.hk/M-Labs/defenestrate/>`__ repository.
+
+Getting started
+---------------
+
+Unless requested otherwise, ARTIQ NixOS NUCs ship with the desktop environment `GNOME <https://www.gnome.org/>`_, a popular choice across many Linux distributions. If you have used Linux before, you're likely already familiar with the interface. A set of basic desktop applications (Firefox, LibreOffice, Gimp) as well as some more specialized tools (Spyder, Jupyter, GTKWave, Wireshark) are also pre-installed and pre-configured. Later you will find that this list of installed packages is precisely specified in the ``configuration.nix`` file, and is very simple to edit to your needs.
+
+.. tip::
+    If you aren't familiar with GNOME, a double-tap of the Windows or Super key will show you the full list of available apps. Plenty of them should be recognizable. It's worth taking particular note of  "Console", which is GNOME's command line terminal emulator, in your case running the `friendly interactive shell <https://fishshell.com/>`_ or ``fish``. From this point forward, the handbook will assume you have a basic familiarity with a Linux environment and commands like ``sudo``. If not, it's probably worth finding some materials on these things before you continue.
+
+The hardware M-Labs ships for pre-installed ARTIQ NUCs is usually the `ASUS NUC 14RVK Pro Kit <https://www.asus.com/Websites/us/products/guxgpgyabgbrsz4o/pdf/6sa39dq3izs4int7.pdf>`_, with Intel i7-155H CPU, 32GB RAM, and a 1GB NVMe. Other options may also be available. Contact ``sales@`` for details.
```

Comment:
I guess better not to put this into the docs, it may change any time without notice